projects/rich-previews/webpack.config.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  node: { fs: 'empty' },
```

Comment:
What does this do?

Reply:
Woops, this was copied from the other config, I am not really sure. I will try to remove it.